index.html

```diff
@@ -222,9 +221,24 @@ <h2>Skills</h2>
             </div>
         </section>
         <!-- End of About -->
-
-    </main>
-    <!-- end of main -->
-</body>
-
+        
+    <section class="contact">
+        <div class="contact-bg-img">
+            <h2>Contact me</h2>
+           <p class="contact-p">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+             I'd love to help with it!</p>
+           <form action="https://formspree.io/f/xjvjejyb" method="POST">
+             <label for="name">Name:</label>
+             <input name="user_name" type="text" id="name" placeholder="Lorem Ipsum" required maxlength="30" >
+             <label for="mail">Email:</label>
+             <input name="user_email" id="mail" type="email" placeholder="loremipsum@gmail.com" required>
+             <label for="msg">Message:</label>
+             <textarea name="user_message" id="msg" placeholder="Write your message here..." maxlength="500" required></textarea>
+             <button type="submit">Get in touch</button>
+            </form>
+            </div>
+            <div class="home-indicator">
+            </div>
+        </section>
```

Comment:
- [ ] Good job creating the contact form, using the correct tags for semantic purposes, and adding all the required validations. However, we have a few minor issues to achieve a great design.

- Kindly align the background image to match the given template

Your design
![image](https://github.com/Srilearns/Portfolio-Website-Mobile-First/assets/81932478/62b990eb-f539-4bc6-a9b1-94baac526656)

Given design
![image](https://github.com/Srilearns/Portfolio-Website-Mobile-First/assets/81932478/9d235511-d551-4053-bd7f-ac597c23c30b)


- [ ] Kindly align the text area to align with the input fields and the `Get In Touch` button so as to match the given design. Hint, you can apply Flexbox to it as you have done for the other elements

Your design
![image](https://github.com/Srilearns/Portfolio-Website-Mobile-First/assets/81932478/19a4a20d-d991-4fb4-b2b6-0aefaa4e7dc8)

Given design
![image](https://github.com/Srilearns/Portfolio-Website-Mobile-First/assets/81932478/b9e560fa-e514-45c6-b002-9d02acc98897)

- [ ] Also add the two background images to the bottom of your design so as to match the given template

Given template
![image](https://github.com/Srilearns/Portfolio-Website-Mobile-First/assets/81932478/5416de2e-d30b-41af-9fd1-446488ddd208)

Your design
![image](https://github.com/Srilearns/Portfolio-Website-Mobile-First/assets/81932478/27fca1e0-b87d-4b6a-afe5-3f8f3e8b7bf3)